api.md

```diff
@@ -170,6 +172,85 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
+
+Newly created distributed hypertables will be able to make use of this
+data node for apportioning data across the distributed database. Note that existing
+distributed hypertables will not automatically use the newly added
+data node. For existing distributed hypertables to use added data
+nodes, use [`attach_data_node`](#attach_data_node).
+
+The function will attempt to bootstrap the data node by:
+1. Connecting to a specified postgres instance
+2. Creating the database that will serve as the new data node
+3. Loading the TimescaleDB extension on the new database
+
+#### Errors
+
+An error will be given if:
+* an attempt is made to run the function inside a transaction _or_
+* the remote database already exists on the data node and
+  `if_not_exists` is not set.
+
+#### Required Arguments [](add_data_node-required-arguments)
+
+| Name        | Description                         |
+| ----------- | -----------                         |
+| `node_name` | Name for the data node.             |
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node. The default is `'localhost'`. |
+| `port`               | Port to use on the remote data node. The default is the PostgreSQL port used by the access node on which the function is executed. |
+| `database`           | Database name where remote hypertables will be created. The default is the current database name. |
+| `password`           | Password to be used when connecting to the remote data node. Note that the user name will be the same as the current user. The default is to not use a password. |
+| `if_not_exists`      | Do not fail if the data node already exists. The data node will not be overwritten, but the command will not generate an error and instead generate a notice. |
+| `bootstrap_database` | Database to use when bootstrapping as described above. The default is `'postgres'`. |
+| `bootstrap_user`     | User to be used for bootstrapping. This user needs to have `SUPERUSER` permissions on the remote database server, unless the remote database already exists with the timescaledb extension. The default is the current user. |
+| `bootstrap_password` | Password to be used when bootstrapping. The default is the password provided in `password`. |
+
+#### Returns [](add_data_node-returns)
+
+| Column              | Description                                       |
+|---------------------|---------------------------------------------------|
+| `node_name`         | Local name to use for the data node               |
+| `host`              | Host name for the remote data node                |
+| `port`              | Port for the remote data node                     |
+| `database`          | Name of database used on the remote data node     |
```

Comment:
```suggestion
| `database`          | Database name used on the remote data node     |
```
Will look more consistent with the above. Otherwise it is good to use article. But I am not so good in English.

Reply:
changed